pkgs/default.nix

```diff
@@ -14477,12 +14477,12 @@ license = stdenv.lib.licenses.mit;
 mkDerivation {
 
 pname = "canonical-json";
-version = "0.5.0.0";
+version = "0.5.0.1";
 src = fetchgit {
 
-url = "https://github.com/well-typed/canonical-json.git";
-sha256 = "19lc5pr85jz3f8ifmjxnkxgib0lz3vgagdny50gb04midc7y37pr";
-rev = "2d261bb971bada1893753b503452d9e6e217bc4a";
+url = "https://github.com/angerman/canonical-json.git";
```

Comment:
So why there and not IOHK?

Reply:
This was before I had access to input-output-hk

https://github.com/input-output-hk/cardano-sl/blob/d91b66c41c753abf2c93fd9cf8ffb976bde5669b/stack.yaml#L144-L148

I'll move it.